backend/de.metas.ui.web.base/src/main/sql/postgresql/system/41-de.metas.ui.web.base/5656820_sys_gh13625_MaterialCockpit_SysConfig.sql

```diff
@@ -2,3 +2,6 @@
 INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,Description,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541482,'O',TO_TIMESTAMP('2022-09-19 19:38:25','YYYY-MM-DD HH24:MI:SS'),100,'Can be overridden on client or org-level','de.metas.material.cockpit','Y','de.metas.ui.web.material.cockpit.field.QtyDemand_SalesOrder.IsDisplayed',TO_TIMESTAMP('2022-09-19 19:38:25','YYYY-MM-DD HH24:MI:SS'),100,'Y')
 ;
 
+-- 2022-09-21T17:05:08.946Z
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,Description,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541484,'O',TO_TIMESTAMP('2022-09-21 20:05:08','YYYY-MM-DD HH24:MI:SS'),100,'Can be overridden on client or org-level','de.metas.material.cockpit','Y','de.metas.ui.web.material.cockpit.field.QtySupply_PurchaseOrder.IsDisplayed',TO_TIMESTAMP('2022-09-21 20:05:08','YYYY-MM-DD HH24:MI:SS'),100,'Y')
+;
```

Comment:
note: looking at this i thought you forgot the other sysconfigs and checked. 
seems to me they are all there in one or other SQL, file 👍